src/HOHQMesh.jl

```diff
@@ -66,7 +70,11 @@ function generate_mesh(control_file;
     flush(tmpio)
 
     # Run HOHQMesh and store output
-    readchomp(`$(HOHQMesh_jll.HOHQMesh()) -f $tmppath`)
+    if ~verbose
+      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -f $tmppath`)
+    else
+      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -verbose -f $tmppath`)
+    end
```

Comment:
```suggestion
    if verbose
      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -verbose -f $tmppath`)
    else
      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -f $tmppath`)
    end
```
That's significantly easier to read for me, but YMMV

Reply:
True I'm not sure why I did it the first way TBH